client/job.lua

```diff
@@ -131,8 +154,12 @@ local function status()
     lib.showContext('ambulance_status_context_menu')
 end
 
---- TODO: Refactor
-local function getPlayerStatus(k, v, result)
+---notify doctor in chat of a patient's health status.
+---TODO: Refactor
+---@param k any
+---@param v any
```

Comment:
v would be table here

Reply:
or an integer in the case of result["BLEED"]. Updated.